packages/webpack/src/ember-webpack.ts

```diff
@@ -213,6 +213,17 @@ const Webpack: PackagerConstructor<Options> = class Webpack implements Packager
       node: false,
       module: {
         rules: [
+          {
+            test: /\.(jpg|jpeg|png|woff|woff2|eot|ttf|svg)$/,
+            use: [
+              {
+                loader: 'url-loader',
```

Comment:
This one is deprecated and replaced by [`asset` loader](https://v4.webpack.js.org/guides/asset-modules/) for webpack 5!